app/controllers/developers_controller.rb

```diff
@@ -46,13 +55,23 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_developer
-      @developer = Developer.find(params[:id])
-    end
 
-    # Only allow a trusted parameter "white list" through.
-    def developer_params
-      params.fetch(:developer, {})
-    end
+  def fetch_master_data
+    @programming_languages = ProgrammingLanguage.pluck(:name, :id)
+    @languages = Language.pluck(:code, :id)
```

Comment:
These instance variables are for the view, right?
Then you put the code, e.g. `ProgrammingLanguage.pluck(:name, :id)`, in the view directly.
Because passing these instance variables don't increase readability or maintanability.
Moreover, it decreases readability. (I have to check two places.).

Refer to the following line.
https://github.com/bbatsov/rails-style-guide#shared-instance-variables
This is ideal, but you shouldn't pass many instance variables to a view.

Reply:
Fixed!